backend/sql.go

```diff
@@ -21,6 +21,151 @@ CREATE TABLE IF NOT EXISTS tokens (
 	created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
 	user_id UUID NOT NULL REFERENCES users(id));
 
+CREATE TABLE students (
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS students (
```

Do this for all tables